src/main/java/frc/robot/subsystems/intake/commands/StartIntake.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+public class StartIntake extends CommandBase {
+    private Intake intake;
+    private boolean direction;
+    public StartIntake(Intake i , boolean direction) {
+        intake = i;
+        this.direction = direction;
+        addRequirements(intake);
+    }
+
+    @Override
+    public void initialize() {
+        if(!intake.isOpen())
+            intake.togglePiston();
+        if(direction)
+            intake.setVelocity(Constants.Intake.VELOCITY);// field --> funnel

```

Comment:
The command should do only one 1 thing, either applying the power or opening the piston